src/components/templates/Jazzbar/JazzTab/JazzTab.tsx

```diff
@@ -121,13 +129,27 @@ const Jazz: React.FC<JazzProps> = ({ setUserList, venue }) => {
   const shouldShowReactions =
     (seatedAtTable && venue.showReactions) ?? DEFAULT_SHOW_REACTIONS;
 
+  const shouldShowJukebox =
+    (!!seatedAtTable &&
+      venue.enableJukebox &&
+      seatedAtTable === MUSIC_VENUE_FIRST_TABLE_NAME) ??
```

Comment:
why only first table?

Reply:
that's the necessary logic, so that only 'driver' table can update the iframe url, that's @sofisparkle 's requirement

Reply:
I believe it's a BM requirement. But for the regular JazzBar venue, it shouldn't be like that?

Reply:
and how is it going to work for BM? 
'driver' is at one table, and other people are at other tables? 
so If I change it at any table it changes for all the tables, correct?